gitkraken/GitKraken.desktop

```diff
@@ -6,5 +6,5 @@ Icon=gitkraken
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;GTK;Development;Documentation;
```

Comment:
This category is from the upstream `.desktop` file, so I want to keep this.
Also this a related category to GTK. https://specifications.freedesktop.org/menu-spec/latest/apas02.html